PhpStorm Protocol (Win)/run_editor.js

```diff
@@ -6,7 +6,12 @@ var settings = {
 	folder_name: 'PhpStorm 9.0',
 
 	// Set to window title (only text after dash sign), that you see, when switching to running PhpStorm instance
-	window_title: 'PhpStorm 9.0'
+	window_title: 'PhpStorm 9.0',
+	
+	// In case your file is mapped via a network share and paths do not match.
+	// eg. /var/www/project_name will be replaced with Y:/project_name
+	project_basepath: '',
+	project_pathalias: ''

```

Comment:
Please rename to `projects_path_alias ` (`s` and `_` added), because we're defining single mapping for all projects, not just one.